src/modules/uniplayer/Uniplayer.jsx

```diff
@@ -99,13 +102,26 @@ export default class Uniplayer extends React.Component {
    this.setState({hoverRange: rangeTime});
  }
 
+ playerActive = () => {
+   this.setState({playerActive: true});
+   this.props.clearTimeouts();
+   if(this.state.playing){
+    this.props.setTimeout(() => {
```

Comment:
If you want to you could promisify this just in case you had code which you wanted await it. Could get complex with the logic that you have which supports multiple timers, and is probably a premature optimization.